radixdlt-java/radixdlt-java/src/main/java/com/radixdlt/client/lib/api/async/AsyncRadixApi.java

```diff
@@ -268,6 +271,16 @@
 			return call(request, new TypeReference<>() {});
 		}
 
+		@Override
+		public Promise<TransactionHistory2> history2(
```

Comment:
Can we write a more descriptive name describing the difference between this method and the existing `history`?